backend-rust/src/indexer.rs

```diff
@@ -122,6 +122,28 @@ SELECT height FROM blocks ORDER BY height DESC LIMIT 1
         .await?
         .map(|r| r.height);
 
+        // Handle TLS configuration and set timeouts according to the configuration for
```

Comment:
Can we move this above `last_height_stored` fetched, since `start_height` below is related to it.